test/functional/rpc_generateblock.py

```diff
@@ -51,37 +51,34 @@ def run_test(self):
         assert_equal(len(block['tx']), 1)
         assert_equal(block['tx'][0]['vout'][0]['scriptPubKey']['addresses'][0], combo_address)
 
-        # Generate 110 blocks to spend
-        node.generatetoaddress(110, address)
+        # Generate some blocks to spend
```

Comment:
I bumped it from 110 blocks because I was getting premature spend when using 110 blocks. I think its because the utxos are sorted by value so their confirmations are somewhat arbitrary in the self._utxos. 200 seemed to be a quick and dirty fix.

Reply:
Yeah, you're right that it's bc the utxos are sorted by value in `MiniWallet` and a lot of them will be immature coin bases. I think what you have is good. But what I've usually done, and seen done before, is generating the utxos you need to spend with `mini_wallet.generate()`, and then generating 100 block with node, `node.generate(100)` so that all the utxos in `MiniWallet` are mature and good to go

Reply:
I pretty much approached with a similar style as described by @mjdietzx in `mempool_limit.py` as well to prevent premature spend error

Reply:
ya that certainly seems like a more coherent approach